internal/server/data/provideruser.go

```diff
@@ -199,7 +261,34 @@ func GetProviderUser(tx ReadTxn, providerID, identityID uid.ID) (*models.Provide
 	return (*models.ProviderUser)(pu), nil
 }
 
-func SyncProviderUser(ctx context.Context, tx WriteTxn, user *models.Identity, provider *models.Provider, oidcClient providers.OIDCClient) error {
+func ProvisionProviderUser(tx GormTxn, user *models.ProviderUser) error {
```

Comment:
I noticed this PR uses `Provision` instead of `Create`. In the RFC they seem to use `Create` as well (https://www.rfc-editor.org/rfc/rfc7644#section-3.3). Are there other scim docs that use `Provision` , or could we use `Create` for the `api` type and these functions?

Reply:
I renamed most of the functions to "Create", but I left the data level as "provision", there is another "CreateProviderUser" function that has different functionality (it relies on the identity already existing)